upgrade-notes/storefront_20260114_123815.md

```
@@ -0,0 +1,13 @@
+#### Add graceful degradation for unsupported browsers ([#4390](https://github.com/shopsys/shopsys/pull/4390))
+
+A dismissible warning banner is now shown for users on unsupported browsers (iOS 12, Safari 12, and other browsers that don't support ES2020 features like optional chaining and nullish coalescing).
+
+#project-base-diff
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)